### DISCUSSION TITLE
More Toggle and options inside visibility

### DISCUSSION THREAD

**Discussion by EzraBlint** [AlistGo/alist#1674] [Q&A]
Contemplating creating a feature request to hide the more toggle the options abaliable. preferaby depending on user logged in.

The reason: 
Accounts like guest (I believe) have no permissions to do anything but read (unless given specific access) unecessary options are still avaliable in the more toggle like upload, make new file or folder etc. If someone like guest tries these functions they only cause errors when used and confusion if visible.

I would be content to hide via CSS but this will then affect accounts like admin so is a less ideal solution.

My skills in CSS are not good and struggling to work out how to hide the toggle. even with various attempts, inspecting elements and (i'll keep persisting and hope I get lucky). 

I thought I would raise a discussion in the meantime to get thoughts and opinions and possibly even help.

**Comment by wengtad** [comment]
Pardon me if I get you wrong. 

You want the guest account to only read?
 It could be set under `Users` settings menu, find the guest account , uncheck everything or selective under `Permission` and `Save`.

This will work if you are using V3.

**Comment by EzraBlint** [comment]
@wengtad thanks for your response, I may have garbled my messgae slightly so apologies if that is the case. As you say it's easy enough to ensure an account only has read permissions. Below is a screnshot to show that.
![Screenshot 2022-09-17 002457](https://user-images.githubusercontent.com/8010518/190830118-587a06f8-3969-406f-885c-486596786c2a.png)

My want is to be able to controll this the options for the below toggle:
![Screenshot 2022-09-17 002444](https://user-images.githubusercontent.com/8010518/190830144-0a61f7a3-0329-4486-a0b3-461954c5a244.png)

be it to hide the entire toggle, be it a manage option or via CSS. This is due to the fact if I have made an account read only I see little point in them being able to attempt to create files/folders, upload or use those errors to work out mount paths.
